File: Data Structures & Algorithms/remove-element/submission-2.py
# Brute Force Solution
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        # Inialize an empty array to temporarily store valid elements 
        tmp = []
        for i in range(len(nums)):
            if nums[i] != val:
                tmp.append(nums[i])
            else:
                continue
        for j in range(len(tmp)):
            nums[j] = tmp[j]

        return len(tmp)

File: Data Structures & Algorithms/concatenation-of-array/submission-2.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = std::size(nums);
        vector<int> ans(2 * n);

        for (int i = 0; i < std::size(nums); i++) {
            ans[i] = nums[i];
            ans[i + n] = nums[i];
        }

       return ans;
    }
};